ztp/siteconfig-generator/siteConfig/testdata/siteConfigDualStackStandardClusterTestOutput.yaml

```diff
@@ -217,6 +217,7 @@ data:
                         - contents:
                             source: data:text/plain;charset=utf-8;base64,IyEvYmluL2Jhc2gKZWNobyAiRm9vYmFyISIgPiBmb29iYXIuY29uZg==
                           mode: 493
+                          overwrite: true
```

Comment:
The MCO will always overwrite any files, but Ignition will not. Setting it to be true should have no impact on ZTP.